dashboard.el

```diff
@@ -25,6 +25,14 @@
 (eval-when-compile (require 'projectile))
```

Comment:
Could `(eval-when-compile (require 'bookmark))` go too?

Does the require at [dashboard.el#L183](https://github.com/andjscott/emacs-dashboard/blob/03c169f3849e691b58101a2c7b7363d8f2ea6f70/dashboard.el#L183) need to be changed to `(eval-when-compile (require 'projectile))`?